config/routes.js

```diff
@@ -24,23 +24,25 @@ module.exports.routes = {
   //  ╚╩╝╚═╝╚═╝╩  ╩ ╩╚═╝╚═╝╚═╝
 
   // No apparent use for webpages right now...
-  '/': { view: 'pages/homepage' },
+  '/':                                    { view: 'pages/homepage' },
 
   //  ╔═╗╔═╗╦  ╔═╗╔╗╔╔╦╗╔═╗╔═╗╦╔╗╔╔╦╗╔═╗
   //  ╠═╣╠═╝║  ║╣ ║║║ ║║╠═╝║ ║║║║║ ║ ╚═╗
   //  ╩ ╩╩  ╩  ╚═╝╝╚╝═╩╝╩  ╚═╝╩╝╚╝ ╩ ╚═╝
 
-  'GET /api/v1/blobs/view': { action: 'blobs/view' },
-  'POST /api/v1/blobs/create': { action: 'blobs/create' },
+  'GET /api/v1/blobs/view':               { action: 'blobs/view' },
+  'POST /api/v1/blobs/create':            { action: 'blobs/create' },
+  'GET /api/v1/blobs/:id':                { action: 'blobs/get-blob' },
+  'GET /share/:id':                       { action: 'blobs/get-blob' },
```

Comment:
Dupe routes?

Reply:
They both do the same thing, `'GET /api/v1/blobs/:id':  ` is more API-ish and `'GET /share/:id':` is something you can share as a link to others. Let me know if there's a better way to do that